pkg/api/dashboard.go

```diff
@@ -106,14 +107,24 @@ func GetDashboard(c *m.ReqContext) Response {
 		meta.FolderUrl = query.Result.GetUrl()
 	}
 
-	isDashboardProvisioned := &m.IsDashboardProvisionedQuery{DashboardId: dash.Id}
-	err = bus.Dispatch(isDashboardProvisioned)
+	provisioningDataQuery := &m.GetProvisionedDashboardDataByIdQuery{DashboardId: dash.Id}
+	err = bus.Dispatch(provisioningDataQuery)
 	if err != nil {
 		return Error(500, "Error while checking if dashboard is provisioned", err)
 	}
 
-	if isDashboardProvisioned.Result {
+	if provisioningDataQuery.Result != nil {
 		meta.Provisioned = true
+		workDir, err := os.Getwd()
```

Comment:
This is probably unclear from the issue. But I think the file path should be relative to the folder specified in the dashboard provisioner, not the current workdir for the binary. 

So if a provisioned is looking at `/tmp/grafana/` and the path to the provisioned file is `/tmp/grafana/web-servers/job-1.json` The modal should show `web-servers/job-1.json`. Any thought on this?

Reply:
Sure I think it makes sense. In case the dashboard dir is outside of the grafana dir this would make  ugly relative paths anyways.

Reply:
I'm not sure it matters if the path is relative to where the dashboard provisioner scans for dashboards.

Reply:
Well it will make the path shorter and I think the main point is to know which file it is rather than the full path. If you did provision the dashboards you probably know where they are. On the other hand having it relative to Grafana working dir could result in paths like `../../../tmp/grafana/dashboard1.json` which I think does not add any value.

Reply:
Hmm ok just realised the dashboard file itself does not have to be inside the config directory :D. Well not sure in that case to be honest. It seems to me that relative to provisioning dir still makes a bit more sense than relative to working dir but it does not seem to be such a strong case as I thought :)

Reply:
We reason for not making it relative to something else then the folder that is being scanned is that it could be considered leaking information about the machine. 

While this might not be super helpful. just showing the filename will give the users some clue about what file to update.